erigon-lib/downloader/downloadercfg/downloadercfg.go

```diff
@@ -203,7 +208,12 @@ func New(dirs datadir.Dirs, version string, verbosity lg.Level, downloadRate, up
 		}
 	}
 	localCfgFile := filepath.Join(dirs.DataDir, "webseed.toml") // datadir/webseed.toml allowed
-	if dir.FileExist(localCfgFile) {
+	exists, err := dir.FileExist(localCfgFile)
+	if err != nil {
+		//TODO: what to do here?
```

Comment:
nothing. error may happen because of "permissions denied" or "disk space is over". just return for user error message (maybe with some context) and stop.

Reply:
done